lib/gems/pending/util/object_storage/miq_swift_storage.rb

```diff
@@ -16,12 +16,15 @@ def initialize(settings)
     @bucket_name = URI(@settings[:uri]).host
 
     raise "username and password are required values!" if @settings[:username].nil? || @settings[:password].nil?
-    _scheme, _userinfo, @host, @port, _registry, @mount_path, _opaque, query, _fragment = URI.split(URI.encode(@settings[:uri]))
```

Comment:
Was the `@mount_path` variable unused before? Seems odd that it was being assigned here if we didn't need it. @jerryk55 ?

Reply:
To clarify, I see that it was being used to set `@container_name`, but because it's an instance variable (rather than a local) I'm being cautious about it being used elsewhere.

Reply:
Just took a closer look. Looks like this is only used in mount sessions (rather than object stores).

Reply:
Yeah, we should probably remove the `@mount_path` and just make it a local variable so it can be garbage collected, but I decided to leave it for now.  Can do a follow up later to fix.

Reply:
Oh, I just looked at the code I wrote, and I did do that... ignore me...